TableBookingAPI/TableBooking.Model/Models/Booking.cs

```diff
@@ -4,7 +4,7 @@ public class Booking : Entity
     {
         public DateTime Date { get; set; }
         public int DurationInMinutes { get; set; }
-        public Table Table { get; set; }
```

Comment:
why you removed navigation of table entity from there? What if you need to get other informations about this Table from this booking other than id in the future?

Reply:
we have navigation to table as property "public Guid TableId { get; set; }" suggest that. We don't need whole Table for our needs so let's keep things simple. As we have Guid TableId we can fetch whole model later in code.